.circleci/config.yml

```diff
@@ -455,54 +436,8 @@ jobs:
       - conditionally-launch-build-and-deploy:
           study_keys: *study_keys
 
-  conditionally-launch-build-and-store-all-job:
-    working_directory: *ng_workspace_path
-    executor:
-      name: build-executor
-    steps:
-      - checkout:
-          path: *repo_path
-      - conditionally-launch-build-and-store:
-          study_key: basil
-      - conditionally-launch-build-and-store:
-          study_key: osteo
-      - conditionally-launch-build-and-store:
-          study_key: brain
-      - conditionally-launch-build-and-store:
-          study_key: angio
-      - conditionally-launch-build-and-store:
-          study_key: mbc
-      - conditionally-launch-build-and-store:
-          study_key: testboston
-      - conditionally-launch-build-and-store:
-          study_key: mpc
-      - conditionally-launch-build-and-store:
-          study_key: atcp
-      - conditionally-launch-build-and-store:
-          study_key: rarex
-      - conditionally-launch-build-and-store:
-          study_key: rgp
-      - conditionally-launch-build-and-store:
-          study_key: esc
-      - conditionally-launch-build-and-store:
-          study_key: prion
-      - conditionally-launch-build-and-store:
-          study_key: cgc
-      - conditionally-launch-build-and-store:
-          study_key: circadia
-      - conditionally-launch-build-and-store:
-          study_key: pancan
-      - conditionally-launch-build-and-store:
-          study_key: singular
-      - conditionally-launch-build-and-store:
-          study_key: brugada
-      - conditionally-launch-build-and-store:
-          study_key: lms
-      - conditionally-launch-build-and-store:
-          study_key: fon
-
   app-build:
-    description: Run build and store build artifacts to Google cloud storage
+    description: Build and store build artifacts to Google cloud storage if existing archive is not found
```

Comment:
Combined steps from `conditionally-launch-build-and-store` command into `app-build` job. Logics to check build archive existence are copied from `conditionally-launch-build-and-store` command. See updated job description for what has changed.